python/overrides.js

```diff
@@ -18,4 +18,8 @@
     // TODO verify and enable
     'appium iOS check fully window with scroll and pageCoverage': { skipEmit: true },
     'appium iOS check window region with scroll and pageCoverage': { skipEmit: true },
+    'appium android landscape mode check window andorid 7': {skip: true},
+    'appium android landscape mode check region android 7': {skip: true},
+    'appium android landscape mode check window andorid 10': {skip: true},
+    'appium android landscape mode check region android 10': {skip: true}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9250217)
```suggestion
    "appium android landscape mode check region android 10": {skip: true}
```